src/drivers/display/nxp_elcdif/lv_nxp_elcdif.c

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file lv_nxp_elcdif.c
+ *
+ * Driver for NXP's ELCD
+ */
+
+#include "lv_nxp_elcdif.h"
+
+#ifdef LV_USE_NXP_ELCDIF
```

Comment:
It's always defined in the worst case  in `lv_conf_internal.h` So it's enough to test `#if LV_USE_NXP_ELCDIF != 0` (`!= 0` is more generic so we usually use it instead of `== 1`)